sycl/source/detail/device_info.hpp

```diff
@@ -1025,13 +1051,27 @@ get_device_info_host<info::device::atomic_memory_order_capabilities>() {
           memory_order::acq_rel, memory_order::seq_cst};
 }
 
+template <>
+inline std::vector<memory_order>
+get_device_info_host<info::device::atomic_fence_order_capabilities>() {
```

Comment:
I wonder when we will be able to stop specializing that for host device, because it doesn't really exists anymore